docs/source/modules.rst

```diff
@@ -1,14 +1,5 @@
-.. initially generated by
-   sphinx-apidoc -f -o source ../ ../versioneer.py ../setup.py
-   (everything after the first "../" are exclude patterns)
-
-PyFstat module documentation
-============================
-
-These pages document the full API and set of classes provided by PyFstat.
-
-See :doc:`here <readme>` for installation instructions and
-other general information.
+PyFstat
+=======
```

Comment:
please reinstate (that's why I suggested to use `git add -p` after `sphinx-apidoc`)

Reply:
:facepalm: I did it a few times correctly. Thanks.